### DISCUSSION TITLE
kanban view doesn't work with link to another record [Oct 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4246] [Discord Threads]
> 🙋‍♂️ **alex.pro**  
##   
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1030921057122656327/Untitled.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### Possibly the reason is different but I see this is the only obvious difference between tables where kanban view works and doesn't work  
#  
> 💁 **o1lab**  
#### <@495466997304262658>  
#  
> 🙋‍♂️ **alex.pro**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.1**  
#  
> 🙋‍♂️ **wkw**  
#### <@939491479599067156> can u share your schema n detailed logs  
#  
> 🙋‍♂️ **wkw**  
#### and did it happen right after u created the kanban view or did you perform other actions?  
#  
> 🙋‍♂️ **wkw**  
#### i tried each LTAR cols n couldnt see the above error  
#  
> 🙋‍♂️ **alex.pro**  
#### Let's have a call? <@495466997304262658>  
#  
> 🙋‍♂️ **alex.pro**  
#### Sent you calendar link in pm  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1030921057445625926)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
